contrib/navidrome.service

```diff
@@ -28,8 +31,3 @@ RestrictRealtime=yes
 SystemCallFilter=~@clock @debug @module @mount @obsolete @privileged @reboot @setuid @swap
 ReadWritePaths=/opt/navidrome/
```

Comment:
Shouldn't this be changed as well?

Reply:
Yep. I should stop trying to code in the middle of the night...